### PR TITLE
Add minimatch resolution for apk file in release task.

### PR DESCRIPTION
**Pull request by shishirx34**

### CODE REVIEW

File: Tasks/google-play-release/task.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
             "label": "APK Path",
             "defaultValue": "",
             "required": true,
-            "helpMarkDown": "Path to the APK file you want to publish to the specified track."
+            "helpMarkDown": "Path to the APK file you want to publish to the specified track. Wildcards can be used. For example, **/*.apk to match the first APK file."
```
Comment (lostintangent) [inline Tasks/google-play-release/task.json]: Just took a look at how the Gradle task formats their glob example, and apparently VSTS supports using back ticks to mark code samples. Can we wrap the **/*.apk span in back ticks to get the same formatting in the UI? Also, can we add ", in any directory." to the end of this description, just to call out what the ** part means.

File: Tasks/google-play-promote/package.json [hunk 2]
```diff
@@ -11,7 +11,6 @@
   "dependencies": {
     "bluebird": "^2.10.1",
     "googleapis": "^2.1.5",
-    "minimatch": "^3.0.0",
```
Comment (lostintangent) [inline Tasks/google-play-promote/package.json]: Maybe you already did it, but can you double check with Peter that vso-task-lib doesn't have a dependency on this? I believe that is why we actually added this to our tasks, even though we don't use it.

**Comment by ryuyu** [comment]
Lgtm

File: Tasks/google-play-release/task.json (hunk 1 above)
Comment (shishirx34) [inline Tasks/google-play-release/task.json]: I have made it _italics_

File: Tasks/google-play-promote/package.json (hunk 2 above)
Comment (shishirx34) [inline Tasks/google-play-promote/package.json]: I checked vso-task-lib and it specifies its own dependency for minimatch. Checked with Peter too.

**Pull request merged**